Makefile

```diff
@@ -1,6 +1,9 @@
+CONTAINER_ENGINE ?= $(shell command -v podman 2> /dev/null || echo docker)
```

Comment:
This doesn't look right to me. At least, the default must `docker-compose` not `docker`.

```
[jkadlcik@zeratul log-detective-website]$ make build-prod
/usr/bin/podman -f docker-compose.prod.yaml build --no-cache
Error: unknown shorthand flag: 'f' in -f
make: *** [Makefile:5: build-prod] Error 125
```

This works though

```
CONTAINER_ENGINE=docker-compose make build-prod
```

Reply:
you are right, I forgot to add `-compose` to it, thanks for noticing :)